### PR TITLE
webpack plugins/components and feed

### PR DESCRIPTION
**Pull request by mike701**

### CODE REVIEW

**Comment by ImmanuelCurrah** [comment]
whoops just seeing this lol

File: TypeScript/webpack.main.config.ts [hunk 1]
```diff
@@ -12,6 +13,16 @@ export const mainConfig: Configuration = {
   module: {
     rules,
   },
+  //can cache groups of components later?
```
Comment (ImmanuelCurrah) [inline TypeScript/webpack.main.config.ts:16]: What does this mean?

File: TypeScript/webpack.rules.ts [hunk 2]
```diff
@@ -28,4 +29,14 @@ export const rules: Required<ModuleOptions>['rules'] = [
       },
     },
   },
+  // {
```
Comment (ImmanuelCurrah) [inline TypeScript/webpack.rules.ts:32]: Do we need this?

**Review by ImmanuelCurrah — APPROVED** [review]

File: TypeScript/webpack.main.config.ts (hunk 1 above)
Comment (mike701) [reply TypeScript/webpack.main.config.ts:16]: so it is a comment for me for webpacks ability to cache groups of components based on basic rule sets. So for example all ux-components or pages etc can be stored together. Which can lead to improvements in performance.

File: TypeScript/webpack.rules.ts (hunk 2 above)
Comment (mike701) [reply TypeScript/webpack.rules.ts:32]: Again a note for myself for a later date really.

**Pull request merged**